parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -1116,6 +1082,50 @@ pub mod pallet {
 				},
 			}
 		}
+
+		pub fn instantiate_milagro_pubkeys(
+			pubkeys: BoundedVec<PublicKey, T::MaxSyncCommitteeSize>,
+		) -> Result<BoundedVec<MilagroPublicKey, T::MaxSyncCommitteeSize>, Error<T>> {
+			let milagro_pubkeys = pubkeys
+				.iter()
+				.map(|bytes| milagro_bls::PublicKey::from_bytes_unchecked(&bytes.0))
+				.collect::<Result<Vec<milagro_bls::PublicKey>, _>>()
+				.map_err(|_| Error::<T>::InvalidSignaturePoint)?;
+			BoundedVec::<MilagroPublicKey, T::MaxSyncCommitteeSize>::try_from(milagro_pubkeys)
+				.map_err(|_| Error::<T>::MilagoG1PointBoundExceeded)
+		}
+
+		pub fn get_participant_pubkeys(
```

Comment:
The convention in rust is that getter functions aren't prefixed with `get_`.

```suggestion
		pub fn participant_pubkeys(
```